portality/api/v1/crud/journals.py

```diff
@@ -5,6 +5,33 @@
 
 
 class JournalsCrudApi(CrudApi):
+
+    @staticmethod
+    def get_journal_swag():
+        template = {
+            "responses": {
+                "200": {
+                    "schema": {}
+                },
+                "400": {
+                    "description": "Bad Request"
+                }
+            },
+            "parameters": [
+                {
+                    "description": "<div class=\"search-query-docs\">DOAJ journal ID. E.g. 4cf8b72139a749c88d043129f00e1b07 .</div>",
+                    "required": True,
+                    "type": "string",
+                    "name": "journal_id",
+                    "in": "path"
+                }
+            ],
+            "tags": ['CRUD']
+        }
+        template['responses']['200']['schema']['title'] = 'Journal schema'
```

Comment:
Good point, I refactored the template into common.py, into the CrudApi class. Might need to move it up to the Api class even. Now all the various endpoints actually share the template, deepcopy it, and add their stuff.